nucleus/cluster/cli/src/main/java/com/sun/enterprise/admin/cli/cluster/RestartLocalInstanceCommand.java

```diff
@@ -66,6 +71,9 @@
     @Param(optional = true, shortName = "d", defaultValue = "false")
     private Boolean debug;
 
+    @Param(optional = true)
+    private int timeout;
```

Comment:
Still got competing timeouts here.
This command calls a RemoteCLICommand which calls CLIRemoteAdminCommand, which is a child of RemoteRestAdminCommand and thus has its own timeout.